src/configeditor/EditBoundingBoxes.tsx

```diff
@@ -0,0 +1,208 @@
+import React from "react";
+import {
+  Input,
+  Button,
+  VStack,
+  FormControl,
+  FormLabel,
+  Text,
+  HStack,
+  FormErrorMessage,
+} from "@chakra-ui/react";
+import { Config } from "../types/config";
+import { BoundingBox } from "../util/bounds";
+import { ErrorObject, useConfigEdit } from "./hooks";
+
+interface LatLngInputProps {
+  title: string;
+  field: string;
+  value: string;
+  error: boolean;
+  onChange: (e: React.ChangeEvent<HTMLInputElement>) => void;
+}
+
+interface EditSingleBoundingBoxesProps {
+  type: string;
+  bounds: BoundingBox | null;
+  values: any;
+  setValues: any;
+  error: boolean;
+}
+
+function LatLngInput(props: LatLngInputProps) {
+  return (
+    <FormControl isInvalid={props.error}>
+      <FormLabel htmlFor={props.field}>{props.title}</FormLabel>
+      <Input
+        id={props.field}
+        type="number"
+        variant="outline"
+        value={props.value}
+        onChange={props.onChange}
+      />
+    </FormControl>
+  )
+}
+
+function EditSingleBoundingBoxes(props: EditSingleBoundingBoxesProps) {
+  const { type, bounds, values, setValues, error } = props;
+
+  const onChange = (value: string, field: "westmost" | "southmost" | "eastmost" | "northmost") => {
+    if (!bounds) {
+      const newBounds = (
+        field === "westmost" ? new BoundingBox(value, "", "", "") :
+        field === "southmost" ? new BoundingBox("", value, "", "") :
+        field === "eastmost" ? new BoundingBox("", "", value, "") :
+        new BoundingBox("", "", "", value)
+      );
+      setValues({
+        ...values,
+        boundingBoxes: {
+          ...values.boundingBoxes,
+          [type]: newBounds.toConfigBbox()
+        }
+      });
+    } else {
+      bounds[field] = value;
```

Comment:
I don't like that this bounding box object is mutated. If the object instance is used in a React component, it won't notice that it changed. BoundingBox is meant as an immutable value object.

BoundingBox could have a setField(field, value) method that returns a _new_ BoundingBox, which is a copy of the old one except for that field.